### ISSUE TITLE
Lodash 4.17.4 is not safe

### ISSUE THREAD

**Issue by shibukawa** [FormidableLabs/babel-plugin-transform-define#46]
https://snyk.io/vuln/npm:lodash:20180130

babel-plugin-transform-define uses 4.17.4 strictly.
Could you update to 4.17.5?

**Comment by tnir** [comment]
Found there is some suspicion even in 4.17.5:

- https://hackerone.com/reports/310443
    - https://github.com/lodash/lodash/commit/d8e069cc3410082e44eb18fcf8e7f3d08ebe1d4a#r28425132

**Comment by pdf13** [comment]
There is a 4.17.10 version
https://www.npmjs.com/package/lodash

**Comment by underblob** [comment]
`npm audit` says `>= 14.7.5`:
```
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ babel-plugin-transform-define [dev]                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ babel-plugin-transform-define > lodash                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 59560 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```
Would be great to get this fixed.

**Comment by MelleB** [comment]
Fixed in PR #52 

Could this PR be merged?

**Comment by ryan-roemer** [comment]
Fixed in https://github.com/FormidableLabs/babel-plugin-transform-define/pull/52
